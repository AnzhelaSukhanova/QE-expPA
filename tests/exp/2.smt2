(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (or (bvult (bvshl #b0001 x) (bvadd (bvmul y #b1011) #b0100)) (= (bvshl #b0001 x) (bvadd (bvmul y #b1011) #b0100)))))
