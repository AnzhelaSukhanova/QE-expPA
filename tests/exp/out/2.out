(let (($e1 (bvadd y #xc))) (let (($e2 (= y #x4))) (let (($e3 (ite (bvult $e1 #x8) (ite (bvult $e1 #x4) (ite (bvult $e1 #x2) (ite $e2 #xf #x0) #x1) #x2) #x3))) (let (($e4 (bvult #xb $e1))) (let (($e5 (bvult #x6 $e1))) (let (($e6 (bvult #xc $e1))) (let (($e7 (bvult #x8 $e1))) (not (and (not $e5) (not $e4) (not $e6) (not $e7) $e2 (not $e4) (not $e5) (not $e6) (not $e7) $e2 (not $e4) (not (bvult (bvshl #xb (bvadd #xe $e3)) $e1)) (not (bvult (bvshl #xb (bvadd #xd $e3)) $e1)) (not $e5))))))))))
